src/panel/direction/RouteSummaryInfo.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import RouteVia from './RouteVia';
+import { Badge, CloseButton, Flex } from 'src/components/ui';
+import { formatDuration, formatDistance } from 'src/libs/route_utils';
+
+const RouteSummaryInfo = ({ route, vehicle, onClose }) =>
+  <div>
+    <Flex alignItems="center" justifyContent="space-between" className="u-text--title">
+      {formatDuration(route.duration)}
+      {onClose && <CloseButton onClick={onClose} />}
```

Comment:
Let's move the button outside, and use `RouteSummaryInfo` itself as a `Flex` child in `MobileRouteDetails`.